### DISCUSSION TITLE
diff update files patch

### DISCUSSION THREAD

**Discussion by 7zxkv** [OnionUI/Onion#1224] [Ideas]
Hi,

sometimes the differences between versions are very small, like between 4.2.1 and 4.2.2. for this kind of version, couldn't we propose a simple file diff-update rather than systematically re-proposing all the files, knowing that updates are quite regular, i'm not sure it's necessary to redo a migration procedure for this kind of very minor version. this is just an opinion, thanks to all.

best regards
 
@Aemiii91 @schmurtzm

**Comment by schmurtzm** [comment]
That's true, I understand. Totofaki was doing small patches with the Onion versions below the 4.0.0 . We choose to avoid that because having multiple patches with diffs is quickly horrible to manage. So we made a installer which is quite simple and almost fast to use 😄 and at least we are sure that the users doesn't have a partial version. Modifying this would require efforts and could give support problems so we prefer to focus on future fun features.
